openapi/openapi-proposal.yaml

```diff
@@ -126,7 +129,14 @@ paths:
       - name: depth
         in: query
         description: The number of eras to query for payouts of. Must be less
-          than or equal to `HISTORY_DEPTH`.
+          than or equal to `HISTORY_DEPTH` for non-archive nodes. Defaults to 1.
```

Comment:
Yeah, agree about the archive thing, oversight on my part.

I am in favor of keeping `era` and removing `at` since I think it would let someone do a query at just a single era without the overhead of trying to figure out a block in the era, which could be non-trivial. e.g. if someone is concerned about response time for the endpoint and they know the exact era, they could just specify depth=1&era=10 to get info just for era 10.